selftest/test-requirements.txt

```diff
@@ -0,0 +1,13 @@
+autoflake
```

Comment:
NIT: Add versioning for the packages.

Reply:
I started with frozen requirements but removed it because the python version used in 20.04 would demand I pretty much keep two sets.  done!